src/main/java/ch/njol/skript/conditions/CondIsStackable.java

```diff
@@ -0,0 +1,36 @@
+package ch.njol.skript.conditions;
```

Comment:
As mentioned in previous comments, you need the license header. You can just grab one from any existing file.